helper/resource/testing_new_import_state.go

```diff
@@ -134,31 +144,31 @@ func testStepNewImportState(ctx context.Context, t testing.T, helper *plugintest
 		}
 	}
 
-	var importWd *plugintest.WorkingDir
+	var workingDir *plugintest.WorkingDir
 
 	// Use the same working directory to persist the state from import
 	if importStatePersist {
-		importWd = wd
+		workingDir = testCaseWorkingDir
```

Comment:
> Personally I don't think this is a big problem, but a little hidden trap is here I just thought about 🤔 

Since we are eventually planning on not applying the plannable import kinds (well the plannable block with ID already has this behavior), this persisted state would have a removed resource from it 😄.

If we remove the apply completely for import by identity, maybe we should just reject any combinations with `ImportStatePersist` (we chatted about this before i think? maybe it's already that way?)

Reply:
Yes, we have it covered 😃 

https://github.com/hashicorp/terraform-plugin-testing/blob/88a9bc9b2c440b6e0f632721e0ca69f583a13c28/helper/resource/testing_new_import_state.go#L430-L431